lib/internal/streams/end-of-stream.js

```diff
@@ -9,8 +9,13 @@ const {
 } = require('internal/errors').codes;
 const { once } = require('internal/util');
 
-function isRequest(stream) {
-  return stream.setHeader && typeof stream.abort === 'function';
+function isLegacyRequest(stream) {
+  return stream.setHeader && typeof stream.abort === 'function' &&
+    typeof stream.writableFinished !== 'boolean';
+}
```

Comment:
When this is going to be triggered? `writableFinished` is missing in IncomingMessage? Has that PR landed yet?

Reply:
As far as I’m aware it has landed.